runtime/lua/vim/lsp/rpc.lua

```diff
@@ -495,10 +495,14 @@ local function start(cmd, cmd_args, handlers, extra_spawn_params)
       decoded.error = convert_NIL(decoded.error)
       decoded.result = convert_NIL(decoded.result)
 
-      -- Do not surface RequestCancelled to users, it is RPC-internal.
-      if decoded.error
-        and decoded.error.code == protocol.ErrorCodes.RequestCancelled then
-        local _ = log.debug() and log.debug("Received cancellation ack", decoded)
+      -- Do not surface RequestCancelled or ContentModified to users, it is RPC-internal.
+      if decoded.error and (decoded.error.code == protocol.ErrorCodes.RequestCancelled
```

Comment:
just `if decoded.error then ` is easier to read and maintain

Reply:
I see, I wasn't sure whether there were other errors that were handled elsewhere. I'll push the change